pkg/installer/upgrade.go

```diff
@@ -58,6 +76,32 @@ func Upgrade(uninstallConfig *apiv1.KubectlStorageOSConfig, installConfig *apiv1
 	return err
 }
 
+// checkForExistingWorkloads ensures no bound PVCs are using a storageos storageclass.
+func (in *Installer) checkForExistingWorkloads() error {
+	pvcList, err := pluginutils.ListPersistentVolumeClaims(in.clientConfig, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	stosSCList, err := pluginutils.ListStorageClasses(in.clientConfig, metav1.ListOptions{LabelSelector: stosAppLabel})
```

Comment:
What is the guarantee that all storageos SC will be labeled with `app=storageos`?

Reply:
Customers are able to create any storageclass what they want, so i think we have to list all of them and filter by provisioner. Thanks